wgengine/wgengine.go

```diff
@@ -49,6 +49,14 @@ type StatusCallback func(*Status, error)
 // NetInfoCallback is the type used by Engine.SetNetInfoCallback.
 type NetInfoCallback func(*tailcfg.NetInfo)
 
+// NetworkMapCallback is the type used by callbacks that hook
+// into network map updates.
+type NetworkMapCallback func(*controlclient.NetworkMap)
+
+// networkMapCallbackHandle is a unique key used to keep track
```

Comment:
```
// A networkMapCallbackHandle is allocated so its pointer address acts
// as a unique map key handle. (It needs to have non-zero size for Go to
// guarantee the pointer is unique.)
```

Reply:
Maybe we just name it `someHandle`?

Reply:
'Some' sounds rather non-descriptive, is that fine?

Reply:
There's not much to describe :) netowrkMapCallbackHandle sounds too specific for what it does. It's so simple we could use it for other handles in the future.

Also: it's not exported, so no API compatibility rules, or even documentation expectations.

Very low bar for naming things when unexported.